File: Modul-3/Soal-4/PRAK304_2410817310012_FAISALTANJUNG.c
#include <stdio.h>
int main(){
    int bilangan;
    printf(" ");
    scanf("%d", &bilangan);

    if(bilangan == 0){
        printf("Nol\n");
    }
    else if( bilangan >= 1 && bilangan < 10 ){
        printf("Satuan\n");
    }
    else if( bilangan >= 11 && bilangan < 20){
        printf("Belasan\n");
    }
    else if( bilangan ==10 || bilangan >=20 && bilangan < 100){
        printf("Puluhan\n");
    }
    else {
        printf("Anda menginput melebihi angka bilangan\n");
    }
}

File: Modul-3/Soal-4/PRAK304_2410817310012_FAISALTANJUNG.py
bilangan = int(input())

if bilangan == 0:
    print("Nol")
elif bilangan >= 1 and bilangan < 10:
    print("Satuan")
elif bilangan > 10 and bilangan < 20:
    print("Belasan")
elif bilangan == 10 or bilangan >= 20 and bilangan < 100:
    print("Puluhan")
else:
    print("Anda menginput melebihi limit bilangan")
